handler/kafkatopic/kafkatopic.go

```diff
@@ -470,23 +470,23 @@ func MessageTimestampTypeValueChoices() []string {
 
 // MinCleanableDirtyRatioOut min.cleanable.dirty.ratio value, source and synonyms
 type MinCleanableDirtyRatioOut struct {
-	Source   SourceType   `json:"source,omitempty"`   // Source of the Kafka topic configuration entry
-	Synonyms []SynonymOut `json:"synonyms,omitempty"` // Configuration synonyms
-	Value    *float64     `json:"value,omitempty"`    // min.cleanable.dirty.ratio
+	Source   SourceType            `json:"source"`             // Source of the Kafka topic configuration entry
+	Synonyms []SynonymOutAltAltAlt `json:"synonyms,omitempty"` // Configuration synonyms
```

Comment:
@byashimov These `AltAltAlt` suffixes seem odd. Could you check code generation?